cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,38 @@
 
 describe('Book Store app', () => {
   before(() => {
+    cy.visit('/login');
+  });
```

Comment:
```suggestion
```
I would remove the hook before, because it is useless and you can move cy.visit into the first test